### PR TITLE
feature/VDX-266

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: packages/oid4vci-demo-frontend/src/configs/banqup-account.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
   "general": {
     "credentialName": "Account credential",
     "issueCredentialType": "Account",
-    "oid4vpAgentBaseUrl": "https://account.banqup.demo.sphereon.com/agent",
-    "oid4vciAgentBaseUrl": "https://account.banqup.demo.sphereon.com/agent",
+    "oid4vpAgentBaseUrl": "https://banqup.demo.sphereon.com/agent",
```
Comment (sanderPostma) [inline packages/oid4vci-demo-frontend/src/configs/banqup-account.json:5]: Already changed in my branch

Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/configs/banqup-account.json:5]: your branch is a side branch. i am not considering those as valid :P, this is now in sync with dev

File: packages/oid4vci-demo-frontend/src/pages/SSILoadingPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    const pageConfig = flowRouter.getPageConfig();
+    const isTabletOrMobile = useMediaQuery({query: '(max-width: 767px)'})
+    const location = useLocation();
+    const {payload, isManualIdentification} = location.state
+
+    useEffect((): void => {
+        // Simulating a verifying process here that navigates to the next step after 10 seconds
+        setTimeout(async (): Promise<void> => {
+            await flowRouter.nextStep({payload, isManualIdentification})
+        }, 10000)
```
Comment (sanderPostma) [inline packages/oid4vci-demo-frontend/src/pages/SSILoadingPage/index.tsx:23]: timeout value from pageConfig? (SSILoadingPageConfig)

**Review by sanderPostma — APPROVED** [review]
Optionally make the 10000 timeout configurable.
For the rest LGTM. We tested this live so it works.

**Pull request merged**